client/lib/js/react/stores/asset_store.js

```diff
@@ -49,26 +48,26 @@ class AssetStore {
 
     onSuccessFetchAssetList(assetList) {
         if (assetList) {
-            const { assets } = assetList;
-
-            if (this.assetMeta.accountToFetch) {
-                if (assets && Object.keys(assets).indexOf('bigchain') > -1) {
-                    this.assetList = assets.bigchain.concat(assets.backlog);
+            const { assets, account } = assetList;
+            if (account && assets) {
+                if (Object.keys(assets).indexOf('bigchain') > -1) {
```

Comment:
Use `assets.hasOwnProperty('bigchain')`, or if you need it to be an array, `Array.isArray(assets.bigchain)`.